packages/twenty-front/src/modules/action-menu/hooks/useActionMenuEntries.ts

```diff
@@ -0,0 +1,28 @@
+import { actionMenuEntriesComponentState } from '@/action-menu/states/actionMenuEntriesComponentState';
+import { ActionMenuEntry } from '@/action-menu/types/ActionMenuEntry';
+import { useSetRecoilComponentStateV2 } from '@/ui/utilities/state/component-state/hooks/useSetRecoilComponentStateV2';
+
+export const useActionMenuEntries = () => {
+  const setActionMenuEntries = useSetRecoilComponentStateV2(
+    actionMenuEntriesComponentState,
+  );
+
+  const addActionMenuEntry = (entry: ActionMenuEntry) => {
```

Comment:
I think we should make more explicit the role of the key in the map
registerActionManuEntries(key, entries[])